src/main/scala/com/github/pjfanning/pekkobuild/PekkoInlinePlugin.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, derived from Akka.
+ */
+
+package com.github.pjfanning.pekkobuild
+
+import sbt.plugins.JvmPlugin
+import sbt._
+import sbt.Keys._
+
+object PekkoInlinePlugin extends AutoPlugin {
+  override def trigger: PluginTrigger = allRequirements
+
+  override def requires: Plugins = JvmPlugin
+
+  private val flagsFor2 = Seq(
```

Comment:
can we call this `flagsForScala2` (and likewise `flagsForScala3`)?

Reply:
Fixed and pushed.